README.rst

```diff
@@ -11,33 +13,51 @@ each using a different `client library`_ to communicate with CrateDB_.
 Prerequisites
 =============
 
-You need install and run CrateDB 0.54.1 or higher before you set things up.
+- You need install and run CrateDB 0.54.1 or higher before you set things up.
```

Comment:
Can we change that to the latest or 5.0.x 🤷 
We don't really support older versions anyway

Reply:
Thanks. Fixed with 79278325.